kong/kong.config.sh

```diff
@@ -170,6 +170,11 @@ addAuthToEndpoint "cron"
 createEndpoint "x509-identity-mgmt" "http://x509-identity-mgmt:3000/api"  '"/x509"' "true"
 addAuthToEndpoint "x509-identity-mgmt"
 
+# service: influx-retriever
+createEndpoint "influxdb-retriever" "http://influxdb-retriever:3000/tss"  '"/tss"' "true"
+addAuthToEndpoint "influxdb-retriever"
+createEndpoint "influxdb-retriever-api-docs" "http://influxdb-retriever:3000/tss/v1/api-docs"  '"/tss/v1/api-docs"' "true"
```

Comment:
I am curious if Kong will or will not ask for a JWT token for this route (`/tss/v1/api-docs`).
My subconscious says that the kong will ask for a token because it is on route `/tss` but I'm not sure. just commenting...

Reply:
I tested. The route is an exception. But I recommend the devops team to check everything. :)

Reply:
I think the kong instruction "addAuthToEndpoint" that forces authentication ... I tested it on kubernetes and authentication was not requested.

Reply:
I understood that thiago's doubt is because the "http://influxdb-retriever:3000/tss" for both of them, he was worried about how kong would treat it.